### PR TITLE
Send address picker postcode in query string instead of request body

### PR DESCRIPTION
**Pull request by ciaran16**

### CODE REVIEW

**Review by superdevx64 — APPROVED** [review]

File: lib/onfido/resources/address.rb [hunk 1]
```diff
@@ -1,10 +1,7 @@
 module Onfido
   class Address < Resource
     def all(postcode)
-      get(
-        path: 'addresses/pick',
-        payload: { postcode: postcode.delete(' ') }
-      )
+      get(path: "addresses/pick?postcode=#{postcode.delete(' ')}")
```
Comment (tjsousa) [inline lib/onfido/resources/address.rb:4]: I think `RestClient` which is used internally supports a `:params` option that would have to be added here but would generally be a cleaner way to pass the query param:
https://github.com/onfido/onfido-ruby/blob/master/lib/onfido/resource.rb#L22-L36

**Comment by tjsousa** [comment]
Would it make sense to update this expectation as well?
https://github.com/onfido/onfido-ruby/blob/c16489a347f88430bf5100f63a7ea2eb792f5cf9/spec/integrations/exceptions_spec.rb#L21

**Comment by ciaran16** [comment]
> Would it make sense to update this expectation as well?
> 
> https://github.com/onfido/onfido-ruby/blob/c16489a347f88430bf5100f63a7ea2eb792f5cf9/spec/integrations/exceptions_spec.rb#L21

@tjsousa Well spotted, I don't think payload has any purpose for all of these tests so I guess I can just remove it?

File: lib/onfido/resources/address.rb (hunk 1 above)
Comment (ciaran16) [reply lib/onfido/resources/address.rb:4]: 👍 I ticketed this up to do as a separate change, as discussed on slack.

**Review by olasitarska — APPROVED** [review]

**Pull request merged**